src/maintainer/knowledge_base.rst

```diff
@@ -1556,3 +1556,16 @@ The way migrations proceed are:
 Sometimes, you might get a migration PR for your package that you don’t want to merge. In that case, you should put that PR in draft status but should never close it.
 If you close the PR, it makes the bot think that another PR implementing the migration is merged instead, letting the migration continue iterating on the graph; however, the downstream dependents fail because the parent (the one we closed the PR of) didn’t really get rebuilt.
 Another reason why it is good to keep the PR open or in draft status is that people might help with it if they want in the future.
+
+Security considerations for conda-forge builds
+==============================================
+
+The public nature of conda-forge’s infrastructure comes with some risks and the measures that we at ``conda-forge`` take to secure our packages are :
+
+1. All the new packages are reviewed by core ``conda-forge`` members before they are accepted to be added as a feedstock. The packages use recipe-generating scripts that work downstream of other trusted repositories (e.g., PyPI, CRAN). 
```

Comment:
This is not true. New packages are reviewed by the `staged-recipes` team which is not a subset of the `core` team.

Also, packages are not required to pull their source from a "trusted repository"; they can be from any publicly available URL. There is some effort by reviewers to check that the URL points to a canonical/official source, but that is not always easy to determine. Anyone can upload to GitHub or PyPI. The download URL could be changed by the maintainer at any time after the first review.

i.e. There is no chain of trust as implied here. There is merely transparency because anyone can inspect the source code used to create a package.

Reply:
Thanks for reviewing @carterbox and pointing these things out. I will update this information.